backend/Squirrel.SqlService/Squirrel.SqlService.WebApi/Controllers/HubController.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.SignalR;
+using Squirrel.Core.BLL.Hubs;
+using Squirrel.Core.BLL.Interfaces;
+using Squirrel.SqlService.BLL.Models.SquirrelHub;
+
+namespace Squirrel.SqlService.WebApi.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class HubController : ControllerBase
+{
+    private readonly IHubContext<SquirrelHub, ISquirrelHubToSend> _hubContext;
+
+    public HubController(IHubContext<SquirrelHub, ISquirrelHubToSend> hubContext)
+    {
+        _hubContext = hubContext;
+    }
+
+
+    [HttpPost("testExecuteQuery")]
+    public async Task<ActionResult>TestExecuteQuery([FromBody] QueryParameters queryParameters)
+    {
+        await _hubContext.Clients.All.TestExecuteQueryAsync(queryParameters.ClientId, queryParameters.FilterName, queryParameters.FilterRowsCount);
```

Comment:
What is this controller about?

Reply:
It was created just for testing. I have already deleted all features which used for testing.

Reply:
Just for testing. Deleted.